style/style.scss

```diff
@@ -0,0 +1,82 @@
+$main-bg: #141d26;
+
+* {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    background: $main-bg;
+}
+
+.keyboard {
+    text-align: center;
+    margin: 0 auto;
+    width: 800px;
+    padding: 13px 0 13px 6px;
+    border-radius: 5px;
+    background: $main-bg;
+    box-shadow: 0 19px 38px rgba(0,0,0,0.30), 0 15px 12px rgba(0,0,0,0.22);
+    user-select: none;
+    transition: 0.5s;
+
+    &__key {
+        height: 45px;
```

Comment:
if your key container is a cube with the same width and height sizes, it's more handy to have variable with size value which you can use in width and height css properties